volttrontesting/utils/platformwrapper.py

```diff
@@ -638,7 +642,7 @@ def _install_agent(self, wheel_file, start, vip_identity):
         if vip_identity:
             cmd.extend(['--vip-identity', vip_identity])
 
-        res = subprocess.check_output(cmd, env=env)
+        res = execute_command(cmd, env=env, logger=_log)
```

Comment:
This should be within try-except block to catch RuntimeError

Reply:
So you mean catch exception and say return None? I would prefer it rather return an exception. Because these are cases that ideally should not happen and hence an exception. Earlier it was throwing CalledProcessError with just return code which was not very useful to debug. So changed it to log the entire command, and the error message and then throwing a RuntimeError